pkg/server/scrub.go

```diff
@@ -25,10 +25,17 @@ func (c *criService) Scrub(i interface{}) fmt.Formatter {
 		p := *o.Process // make a copy of the process
 		p.Env = []string{_scrubbedReplacement}
 		o.Process = &p
+		o.Annotations = map[string]string{_scrubbedReplacement: _scrubbedReplacement}
```

Comment:
Could probably have a `var scrubbedMap = map[string]string{_scrubbedReplacement: _scrubbedReplacement}` at the top of this package and just assign them all to this instead of making a new map each time